build-tools-internal/version.properties

```diff
@@ -38,7 +38,7 @@ httpclient        = 4.5.13
 httpcore          = 4.4.13
 httpasyncclient   = 4.1.5
 commonslogging    = 1.1.3
-commonscodec      = 1.14
+commonscodec      = 1.15
```

Comment:
I wonder if we should only upgrades those dependencies for the module only? This is going to impact many projects/modules. @rjernst do you have an opinion?

Reply:
Usually, we update the transitive dependencies when we do library upgrades, but we can also don't do it in this case because `jackson` and `commonscodec` updates are really minor.

Reply:
I looked at release notes for jackson 2.3.13 and commons codec 1.15 and saw nothing outstanding. There are changes around Base64 and MurmurHash3 but IIUC existing behaviors are not changed.